### PR TITLE
fix: model version name overflow on mobile [ET-384]

### PR DESCRIPTION
**Pull request by EmilyBonar**
## Ticket

[ET-384]

## Test Plan

Go to a Model Version page, shrink your viewport to ~mobile width, and confirm that the buttons (including overflow button) are present.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[ET-384]: https://hpe-aiatscale.atlassian.net/browse/ET-384?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/pages/ModelVersionDetails/ModelVersionHeader.tsx [hunk 1]
```diff
@@ -155,14 +156,14 @@ const ModelVersionHeader: React.FC<Props> = ({
...
       <div className={css.headerContent}>
-        <div className={css.mainRow}>
-          <div className={css.title}>
+        <Row justifyContent="space-between" wrap>
+          <div className={css.mainRow}>
             <div className={css.versionBox}>V{modelVersion.version}</div>
-            <h1 className={css.versionName}>
-              {modelVersion.name ? modelVersion.name : `Version ${modelVersion.version}`}
-            </h1>
+            <Title truncate={{ tooltip: true }}>
```
Comment (keita-determined) [inline webui/react/src/pages/ModelVersionDetails/ModelVersionHeader.tsx:162]: Do we wanna hide the version name?

### Before

image


### After

image

Comment (EmilyBonar) [reply webui/react/src/pages/ModelVersionDetails/ModelVersionHeader.tsx:162]: No, good catch

Comment (keita-determined) [reply webui/react/src/pages/ModelVersionDetails/ModelVersionHeader.tsx:162]: i dont see any changes.

Comment (EmilyBonar) [reply webui/react/src/pages/ModelVersionDetails/ModelVersionHeader.tsx:162]: It's there now

**Review by keita-determined — APPROVED** [review]

**Pull request merged**